packages/agent/src/sync-engine-level.ts

```diff
@@ -132,27 +134,18 @@ export class SyncEngineLevel implements SyncEngine {
       }
 
       const replyEntry = reply.entry;
-
```

Comment:
This was only syncing `RecordsWrite` messages, added test path for `ProtocolsConfigure` message and simplified the logic here.